src/logistration/RegistrationFailure.jsx

```diff
@@ -2,48 +2,61 @@ import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { Alert } from '@edx/paragon';
+import { INTERNAL_SERVER_ERROR } from './data/constants';
 
-const hasNoErrors = (userErrors) => (
-  userErrors.every((errorList) => (!errorList[0]))
+const showAlert = (userErrors) => (
+  <Alert variant="danger">
+    <Alert.Heading className="text-danger">
```

Comment:
We don't need the class `text-danger` as UX has suggested we use the same color heading and body text.